Branch-SDK/src/main/java/io/branch/referral/Branch.java

```diff
@@ -590,24 +591,19 @@ public static Branch getInstance() {
      */
     public static Branch getInstance(@NonNull Context context, @NonNull String branchKey) {
         if (branchReferral_ == null) {
-            branchReferral_ = Branch.initInstance(context);
-        }
-        if (branchReferral_.prefHelper_.isValidBranchKey(branchKey)) {
```

Comment:
just some cleanup, the method was potentially resetting branch key at run time which the method was not meant to do.